src/monodroid/jni/debug.cc

```diff
@@ -214,13 +212,11 @@ Debug::start_debugging_and_profiling ()
 	androidSystem.monodroid_get_system_property (Debug::DEBUG_MONO_CONNECT_PROPERTY, &connect_args);
 
 	if (connect_args != nullptr) {
```

Comment:
Just another thought, could `connect_args` sometimes be an empty string here, too?

Reply:
In theory it could, since we might read its value from the environment file